.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      - migrate-tests-to-actions
```

Comment:
I'll just remove this part before merging. It was probably just for testing. Is that right?

```suggestion
```

Reply:
Thanks!!! Whoops :)